lib/travis/api/v3/queries/beta_migration_request.rb

```diff
@@ -1,13 +1,39 @@
 module Travis::API::V3
   class Queries::BetaMigrationRequest < Query
+    class ApiRequestFailed < StandardError; end
```

Comment:
@pavel-d I would recommend you name this as `ComApiRequestFailed` to make it clearer.